master/buildbot/newsfragments/unauthorized-login.bugfix

```diff
@@ -0,0 +1 @@
+Fixed UnauthorizedLogin errors during buildbot restart. Fixes (:issue:`3462`).
```

Comment:
I think warn here that there is a new lock.

    Worker authentication is now delayed via a DeferredLock until Buildbot configuration is finished. This fixes UnauthorizedLogin errors during buildbot restart (:issue:`3462`)

Reply:
I updated the news fragment accordingly :+1: